helix-view/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub mod tree;
 pub mod view;
 
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Default, Debug)]
-pub struct DocumentId(usize);
+pub struct DocumentId(pub usize);
```

Comment:
Rather than exposing the internal `usize`, implement `Display` for the type that prints the underlying number.